app/views/dashboard/visit_groups/_edit_visit_form.html.haml

```diff
@@ -22,21 +22,25 @@
   = hidden_field_tag :service_request_id, service_request.id
   = hidden_field_tag :sub_service_request_id, sub_service_request.id
   .form-group
-    = a.label "name", t(:dashboard)[:visit_groups][:name], class: "col-sm-4 control-label"
+    = a.label :name, t(:dashboard)[:visit_groups][:name], class: "col-sm-4 control-label required"
     .col-sm-7
-      = a.text_field "name", {class: 'form-control'}
+      = a.text_field :name, {class: 'form-control'}
   .form-group
-    = a.label "day", t(:dashboard)[:visit_groups][:day], class: "col-sm-4 control-label"
+    = a.label :day, t(:dashboard)[:visit_groups][:day], class: "col-sm-4 control-label required"
     .col-sm-7
-      = a.text_field "day", {class: 'form-control', maxlength: 4}
+      = a.text_field :day, {class: 'form-control', maxlength: 4}
   .form-group
-    = a.label "window_before", t(:dashboard)[:visit_groups][:window_before], class: "col-sm-4 control-label"
+    = a.label :position, t(:dashboard)[:visit_groups][:position], class: "col-sm-4 control-label required"
     .col-sm-7
-      = a.text_field "window_before", {class: 'form-control'}
+      = a.select :position, options_from_collection_for_select(arm.visit_groups.where.not(id: visit_group.id), 'position', 'insertion_name', visit_group.position) + content_tag(:option, t(:visit_groups)[:prompts][:add_as_last], value: "#{arm.visit_groups.count+1}"), { include_blank: t(:constants)[:prompts][:none] }, class: "form-control selectpicker"
```

Comment:
The code generating the options for the select should be pulled out into a helper.